Tests/LegacyUnit/BagBuilder/AbstractTest.php

```diff
@@ -520,6 +520,73 @@ public function testWhereClauseCanSelectPids()
         );
     }
 
+    public function testWhereClausePartGetKeyMustNotBeEmpty()
+    {
+        $this->expectException(
+            \InvalidArgumentException::class
+        );
+        $this->expectExceptionMessage(
+            'The parameter $key must not be empty.'
+        );
```

Comment:
In the new tests, could you please reformat those statements to be in one line only? Thanks!

Ditto for the other tests.